tasks/unix/install-github-release.yml

```diff
@@ -40,6 +40,8 @@
         dest: "{{ ccache_work_dir.path }}"
         mode: "0755"
         remote_src: true
+      environment:
+        PATH: "{{ ansible_pkg_mgr_path }}"  # Needed on Mac to use tar from Homebrew
```

Comment:
"to use GNU tar" would be more accurate than "to use tar from Homebrew", since it might come from Mac Ports

Reply:
Sure. Fixed in 7411451.